### PR TITLE
snippets added

### PR DESCRIPTION
**Pull request by PierreFontaine**
Basics stuff

### CODE REVIEW

File: snippets/dart.cson [hunk 1]
```diff
@@ -13,7 +13,58 @@
...
+
+  'print':
+    'prefix': 'print'
+    'body': """
+      print($1);
+      """
+  'for':
+    'prefix': 'for'
+    'body': """
+      for(var i = 0; i < count; i++){
```
Comment (devoncarew) [inline snippets/dart.cson]: nit: `for (` (space between the `for` and the `(`

File: snippets/dart.cson [hunk 2]
```diff
@@ -13,7 +13,58 @@
...
+        //doSomeStuff
+      }
+      """
+  'while':
+    'prefix': 'while'
+    'body': """
+      while($1){
```
Comment (devoncarew) [inline snippets/dart.cson]: nit: `while (`

File: snippets/dart.cson [hunk 3]
```diff
@@ -13,7 +13,58 @@
...
+    'body': """
+      if($1){
+
       }
 
       """
+  'ifElse':
+    'prefix': 'ifElse'
+    'body': """
+      if($1){
```
Comment (devoncarew) [inline snippets/dart.cson]: ==> `if ($1) {` (space between `)` and `{`

**Comment by devoncarew** [comment]
Thanks for the PR! Can you update to add spaces between keywords and parans (`if(` ==> `if (`) and parans and braces (`){` ==> `) {`)?

Also, it'd be great to see a screenshot of it in action in the PR. Thanks!

**Comment by PierreFontaine** [comment]
image

Screenshot for the latest version.

**Comment by devoncarew** [comment]
Thanks! lgtm

**Pull request merged**